soil/__init__.py

```diff
@@ -142,6 +143,9 @@ def set_progress(cls, task, current, total):
                 task.update_state(state='PROGRESS', meta={'current': current, 'total': total})
         except (TypeError, NotImplementedError):
             pass
+        except IntegrityError:
+            # Not called in task context just pass
+            pass
```

Comment:
`<@task: corehq.apps.ota.tasks.prime_restore>`

seems to be the function that the `@task` decorator is wrapping